docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
 
 ![without policy URL](images/without_policy_URL.png)
-![with policy URL](images/with_policy_URL.png)
 
-Format: `policy INDEX`
+**Format**: `policy INDEX`
 
 * Selects the client at the specified `INDEX`.
 * The index refers to the index number shown in the displayed client list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* Policy ID cannot contain '>' character!
-* Note that your URLs with should not contain '>' characters either! They are not valid URLs by the Internet's definition.
+* The index must be more than 1, and less than or equal to the index of the last item in the displayed list.
 
-Examples:
+**Examples**:
 * `list` followed by `policy 2` displays the policies associated with the 2nd person in the address book.
-* `find Betsy` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
+  
+  ![with policy URL](images/with_policy_URL.png)
+  
+* `find n/Bernice` followed by `policy 1` displays the policies associated with the 1st person in the results of the `find` command.
+  
+  ![with policy URL](images/find-then-policy.png)
+
+[Return to Table of Contents](#table-of-contents)
+
 
-### Sorting the list of clients : `sort`
+### Delete client: `delete`
 
-Sorts the list of clients in ClientBook.
+**Purpose**: Deletes the specified client from ClientBook after listing or finding contacts.
 
-Format: `sort ATTRIBUTE DIRECTION`
+**Format**: `delete INDEX`
+
+* Deletes the client at the specified `INDEX`.
+* The index refers to the index number shown in the displayed client list.
+* The index must be more than 1, and less than or equal to the index of the last item in the displayed list.
+
+**Examples**:
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
+
+  ![delete 2](images/delete-2.png)
+
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+
+  ![delete charlotte](images/delete-charlotte.png)
+
+[Return to Table of Contents](#table-of-contents)
+
+
+### Sort list of clients: `sort`
+
+**Purpose**: Sorts the current list of clients in ClientBook.
+
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
 * Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
+* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy, but not both.
+* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+**Examples**:
+* Sort the current list of clients by name in **ascending** order.
+  * `sort -n -asc`
```

Comment:
Aside from the example on sorting by name, do you think an example on sorting insurance policy is needed?